FFMpegCore.Test/Resources/TestResources.cs

```diff
@@ -21,5 +21,9 @@ public static class TestResources
         public static readonly string PngImage = "./Resources/cover.png";
         public static readonly string ImageCollection = "./Resources/images";
         public static readonly string SrtSubtitle = "./Resources/sample.srt";
+        public static readonly string AiffAudio = "./Resources/sample3aiff.aiff";
```

Comment:
Could you resize or shorten the `sample3aiff.aiff` test file?

Reply:
To maybe around 2-3mb or less

Reply:
Done